### PR TITLE
plan: Clean up

### PR DESCRIPTION
**Pull request by zimulala**
Variables are declared or assigned when it's used.
Rename `Ts` to `Tbl` like `Idx`.

### CODE REVIEW

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -224,32 +217,38 @@ func (p *DataSource) convert2IndexScan(prop *requiredProperty, index *model.Inde
...
-			if !memDB && client.SupportRequestType(kv.ReqTypeIndex, 0) {
-				indexConditions, tableConditions := detachIndexFilterConditions(newSel.Conditions, is.Index.Columns, is.Table)
-				is.IndexConditionPBExpr, is.indexFilterConditions, indexConditions = expressionsToPB(sc, indexConditions, client)
-				is.TableConditionPBExpr, is.tableFilterConditions, tableConditions = expressionsToPB(sc, tableConditions, client)
-				newSel.Conditions = append(indexConditions, tableConditions...)
-			}
+		if client != nil && !infoschema.IsMemoryDB(p.DBName.L) &&
+			client.SupportRequestType(kv.ReqTypeIndex, 0) {
+			idxConds, tblConds := detachIndexFilterConditions(newSel.Conditions, is.Index.Columns, is.Table)
+			is.IndexConditionPBExpr, is.indexFilterConditions, idxConds = expressionsToPB(sc, idxConds, client)
```
Comment (hanfei1991) [inline plan/physical_plan_builder.go]: It's better to rename idxConds to idxRestConds.

File: plan/physical_plan_builder.go [hunk 2]
```diff
@@ -224,32 +217,38 @@ func (p *DataSource) convert2IndexScan(prop *requiredProperty, index *model.Inde
...
+			is.TableConditionPBExpr, is.tableFilterConditions, tblConds = expressionsToPB(sc, tblConds, client)
```
Comment (hanfei1991) [inline plan/physical_plan_builder.go]: It's better to rename tblConds to tblRestConds.

**Comment by hanfei1991** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**